src/spikeinterface/core/tests/test_unitsaggregationsorting.py

```diff
@@ -66,31 +72,46 @@ def test_unitsaggregationsorting(create_cache_folder):
     assert "area" not in sorting_agg_prop.get_annotation_keys()
     assert "date" not in sorting_agg_prop.get_annotation_keys()
 
+
+def test_unitsaggregationsorting_properties(three_sortings):
+    """Aggregates a sorting and check if properties were correctly propogated."""
+
+    sorting1, sorting2, sorting3 = three_sortings
+    num_units = sorting1.get_num_units()
     # test properties
 
-    # complete property
+    # Can propogate property
     sorting1.set_property("brain_area", ["CA1"] * num_units)
     sorting2.set_property("brain_area", ["CA2"] * num_units)
     sorting3.set_property("brain_area", ["CA3"] * num_units)
 
-    # skip for inconsistency
-    sorting1.set_property("template", np.zeros((num_units, 4, 30)))
-    sorting1.set_property("template", np.zeros((num_units, 20, 50)))
-    sorting1.set_property("template", np.zeros((num_units, 2, 10)))
-
-    # incomplete property (str can't be propagated)
-    sorting1.set_property("quality", ["good"] * num_units)
-    sorting2.set_property("quality", ["bad"] * num_units)
+    # Can propogate, even though the dtype is different, since dtype.kind is the same
```

Comment:
```suggestion
    # Can propagate, even though the dtype is different, since dtype.kind is the same
```
@chrishalcrow did you do this on purpose??? :)